src/test/java/io/nats/client/NatsTestServer.java

```diff
@@ -208,34 +224,22 @@ public void start() {
         if (this.customArgs != null) {
             cmd.addAll(Arrays.asList(this.customArgs));
         }
-            
-        if (debug) {
-            cmd.add("-DV");
-        }
+
+        cmd.add("-DV");
 
         this.cmdLine = String.join(" ", cmd);
 
         try {
             ProcessBuilder pb = new ProcessBuilder(cmd);
 
-            if (debug) {
-                System.out.println("%%% Starting [" + this.cmdLine + "] with redirected IO");
-                pb.inheritIO();
-            } else {
-                String errFile = null;
-                String osName = System.getProperty("os.name");
-
-                if (osName != null && osName.contains("Windows")) {
-                    // Windows uses the "nul" file.
-                    errFile = "nul";
-                } else {                
-                    errFile = "/dev/null";
-                }
-
-                pb.redirectError(new File(errFile));
-            }
+            pb.redirectErrorStream(true);
+            pb.redirectError(errorRedirector);
+            pb.redirectOutput(outputRedirector);
+            LOG.info("%%% Starting [" + this.cmdLine + "] with redirected IO");
 
             this.process = pb.start();
+
+            NatsOutputLogger.logOutput(LOG, this.process, NATS_SERVER, port);

```

Comment:
I've redirected logging to a log handler that logs nats-server stdout/stderr at info logging levels, this way one just needs to add the `--info` flag to gradle in order to see nats-server output when running/debugging tests.